prime/script/deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ ! -f prime/script/deploy.sh ]; then
+    echo "Run this script from project root dir (ostelco-core)"
+    exit 1
+fi
+
+export PROJECT_ID="$(gcloud config get-value project -q)"
+export PRIME_VERSION="$(gradle prime:properties -q | grep "version:" | awk '{print $2}' | tr -d '[:space:]')"
+export BRANCH_NAME=$(git branch | grep \* | cut -d ' ' -f2)
+
+echo PROJECT_ID=${PROJECT_ID}
+echo PRIME_VERSION=${PRIME_VERSION}
+echo BRANCH_NAME=${BRANCH_NAME}
+
+prime/script/check_repo.sh ${BRANCH_NAME}
```

Comment:
Being pedantic, I would prefer testing the existence of this script near the top  of the file and bailing out if it doesn't exist. Also I'd put it in a variable and say something like

   $CHECK_REPO ${BRANCH_NAME}

... but again, this is just me being pedantic.